### ISSUE TITLE
[Feature] Quest / Bounty Gamification

### ISSUE THREAD

**Issue by fsawa-san** [stashapp/stash-box#569]
**Is your feature request related to a problem? Please describe.**
The requirements of meta data, which is mandatory often blocks or denies contributions to StashDB. This leads to the following issues:

- It's difficult or in some cases even impossible to contribute, since meta data can sometimes not be retrievable or need extensive research to be found.
- There are some studios, which at the current state, probably will never enter StashDB, even though other valid and correct meta data is available at this time. This will lead eventually to even further meta data loss, when that studio for example closes or decides to tinker with the current correct meta data.
- Sometime wrong metadata will be filled out knowingly (ie. There are several hundred pages with Release Date = 1970-03-01 or Scenes with missing Actors). This leads to an even greater data inconsistency problems, since it will be really difficult to find, which data was filled out wrong. 
- It discourages contribution overall, even if correct and valid meta data is available.
- A lot of duplicate work / effort on contributors side. (ie. 3 separate contributors try to add scenes for studio X, all have to do the same work / effort / research, to come to the conclusion that it is currently not possible to find the required meta data, therefore entering the scenes to StashDB is currently impossible)

**Describe the solution you'd like**

1. Create a value which works as a placeholder for the missing meta data. For Example: 'Release Data = Open Quest, Actor = Open Quest'. This value would work as an indicator that this meta data is still open and needs work.
2. Create a Navigation Tab 'Quest'. When clicking it, you will receive one scene, which has an 'Open Quest'. Try to solve the quest or abandon it, if it is not solvable - to receive a new one.
3. When solving a quest - the changes goes into the standard change request queue.
4. When abandoning a quest - add a 'Abandoning Count' on the scene, the more abandoning counts the less likely a scene will appear as a quest for an other contributor. The idea is, when meta data for a scene has been checked multiple times, then it may be lost or difficult to find. It also spaces the frequency of check for meta data availability . After a certain 'Abandoning Count' threshold the meta data field could go from 'Open Quest' to 'Failed Quest / Meta Data lost' - this would be a state, where it is given up and the meta data is assumed to be lost, at this state it will not show up in any Quest Solving Queue.

Also never give the same Quest to the same contributor, when it was abandoned before and never give a contributor his own quests. To prevent further abuse / lazy quest opening, make opening a quest cost 'Quest Currency', which can be earned by contributing scenes and / or by completing quests.

Additionally on Performers / Studio Pages open quests could be shown, which would make it visible at a glance if meta data is missing or 100% complete.

**Comment by ghost** [comment]
I agree with marking a field as missing but I don't agree about the gamification aspect. In a few years AI will do all the curating so there is no point for that.

**Comment by fsawa-san** [comment]
"In a few years AI will do all the curating so there is no point for that."

I'm not exactly sure if this is a sound argument. If AI will do all the curating then there is no point in the current form of StashDB at all, it basically encourages not contributing and just waiting.

**Comment by ghost** [comment]
> there is no point in the current form of StashDB at all

You still need to keep a database of the metadata in case a website shuts down.

Related to #417

**Comment by Leopere** [comment]
We have a lot of scrapers currently plucking away from the various sources that feed quality data into the API, however, yes I do see the more stringent the requirements for a submission to pass the harder it becomes to play the game.  I realize that game-ification is a touchy subject but it could be done in ways that turn the engagements up from just "Hey vote for or against this edit" to it could be some kind of a two or more, player game possibly based on trivia or something interesting.  The end goal is obviously to help ensure that a real human with the capabilities not yet developed in AI/ML even Google knows that one of the best ways to dial in a dataset is human-augmented ML because generative algorithms can spend thousands of generations finding their way down bad paths.

I may not entirely agree with the first post as for its reasons but I can see why it would be desireable to add some kind of game-ification as a dimension for ensuring properly curated data.  Not just in scoreboard format but like if we coulld find a way to provide actually desireable entertainment along side the database improvements that could be pretty sweet imo.
